src/test/features/constants.tact

```diff
@@ -46,6 +49,18 @@ contract ConstantTester {
         return self.something8;
     }
 
+    get fun something9(): Int {
+        return self.something9;
+    }
+
+    get fun something10(): Int {
+        return self.something10;
+    }
+
+    get fun something11(): Int {
+        return self.something11;
+    }
+
```

Comment:
Looks like we are missing tests for this new part

Reply:
these aren't tested in this version of `main`, as `feature-constants.spec.ts` was added in #237. I will rebase and update this PR now.